### DISCUSSION TITLE
Shred- how many times does it over write?

### DISCUSSION THREAD

**Discussion by PDA123** [bleachbit/bleachbit#1740] [Q&A]
When using Shred- how many times does bleachbit overwrite a file before it finally deletes it?

Where are the parameters for Shred located?

I'm running Debian 12

**Comment by az0** [comment]
Please see [this documentation](https://docs.bleachbit.org/doc/shred-files-and-wipe-disks.html)

**Reply by PDA123** [reply]
Uh oh.  Here's a problem- I shred files using Bleachbit and then using "locate" in command line- those file names still are presented. Yes,the files are gone BUT...the file names persist.

Scary

**Reply by PDA123** [reply]
Found out that BB doesn't delete something-or-another "data-base" where files names are located or somethin'.

Solved by using in command line- updatedb

So much for security and privacy.
